.eslintrc

```diff
@@ -7,6 +7,7 @@
   "rules": {
     "flowtype/no-weak-types": 0,
     "flowtype/require-parameter-type": 0,
-    "flowtype/require-return-type": 0
+    "flowtype/require-return-type": 0,
+    "id-length": 0
```

Comment:
Create a new file `/test/.eslintrc` and add `{"extends": "canonical/ava", "rules": {"id-length": 0}}`.

While you are here, please add `{"root": true}` and remove `canonical/ava` from `/.eslintrc`.